### DISCUSSION TITLE
Point Blank method

### DISCUSSION THREAD

**Discussion by CrazyDibber** [3kh0/ext-remover#447] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Hi, just wondering how would I do point blank (old) on an admin installed extension if it doesn't have a webstore page?

**Comment by obkduishinijf** [comment, marked as answer]
it doesn't need to have a webstore page, you go to something that wasn't installed by admin, like google docs offline, go to the details, webstore, and press esc before it loads, then you go to your admin installed extension, copy the id
for example: on something like goguardian's, chrome://extensions/?id=haldlgldplgnggkjaafhelgiaglafanh is what the link is, you copy everything after ?id=
then go back to the unloaded webstore page, run the bookmarklet (or javascript in the omnibox, nerd) and paste in the extension id. now it should work.

**Comment by Aka-but-nice** [comment]
> Hi, just wondering how would I do point blank (old) on an admin installed extension if it doesn't have a webstore page?

You don't do (old) point-blank on a admin extension webstore page, you have to use (old) point-blank on a extension webstore page you installed.

**Comment by Crazygird** [comment]
I just realized that you can use any window.open link to chaeat.

**Reply by Aka-but-nice** [reply]
Yea, but you can use directly window.open() on a chrome page, that's not how it works, chrome pages have extra security, even though on settings there is a link that straight up opens a about:blank page.

**Comment by Crazygird** [comment]
This thread has been resolved.
